AdyenCardScannerTests/CardImageParserTests.swift

```diff
@@ -0,0 +1,153 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@testable import AdyenCardScanner
+import XCTest
+
+
+final class CardImageParserTests: XCTestCase {
+
+    var sut: CardImageParser!
+
+    override func tearDownWithError() throws {
+        sut = nil
+        try super.tearDownWithError()
+    }
+
+    func testParseImageWithHighContrastCardImage() throws {
+        // Given
+        let expirationDateFormatter = ExpirationDateFormatter()
+        sut = CardImageParser(expirationDateFormatter: expirationDateFormatter)
+
+        let testCreditCard = try XCTUnwrap(testCreditCardHighContrast)
+        let expectation = expectation(description: "Image should be parsed")
+
+        // When
+        sut.parse(image: testCreditCard.image) { receivedCreditCard in
+            expectation.fulfill()
+
+            let expectedCreditCard = testCreditCard.creditCard
+            XCTAssertEqual(expectedCreditCard.number, receivedCreditCard.number)
+            XCTAssertEqual(expectedCreditCard.expirationDate, receivedCreditCard.expirationDate)
+        }
+
+        // Then
+        waitForExpectations(timeout: 10.0)
```

Comment:
We have added `wait` api in  `XCTestCase extension`. Should we use that one instead of using `waitForExpectations` ?